ui/component/fileDetails/view.jsx

```diff
@@ -113,11 +113,11 @@ class FileDetails extends PureComponent<Props> {
 }
 // move this with other helper functions when we re-use it
 function formatBytes(bytes, decimals = 2) {
-  if (bytes === 0) return '0 Bytes';
+  if (bytes === 0) return __('0 Bytes');
 
   const k = 1024;
   const dm = decimals < 0 ? 0 : decimals;
-  const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
+  const sizes = [__('Bytes'), __('KB'), __('MB'), __('GB'), __('TB'), __('PB'), __('EB'), __('ZB'), __('YB')];
```

Comment:
Maybe `Bytes` should be wrapped in `__()`, but I don't think the rest of these should be

Reply:
Well, it seems they should. For example im my language or in French MB is translated as MO (from mega octet) so KB, MB, GB etc will me KO, MO, GO etc.

Reply:
Ok. I didn't know that! I think we should keep it as is. Can you rebase this PR? Then it's ready to merge.